pkg/apis/serving/fieldmask.go

```diff
@@ -596,12 +596,12 @@ func SecurityContextMask(ctx context.Context, in *corev1.SecurityContext) *corev
 
 	// Allowed fields
 	out.RunAsUser = in.RunAsUser
+	out.RunAsNonRoot = in.RunAsNonRoot
```

Comment:
This does not quite do what the issue claimed: We wanted to allow `runAsNotRoot=true` unconditionally, not all possible settings IIRC.

Reply:
Addressed this in the subsequent commit.  Please let me know if I am missing any other case.